soco/core.py

```diff
@@ -1833,6 +1873,185 @@ def add_item_to_sonos_playlist(self, queueable_item, sonos_playlist):
             ('AddAtIndex', 4294967295)
         ])
 
+    def reorder_sonos_playlist(self, object_id, tracks, new_pos, update_id=0):
+        """ Reorder and/or Remove tracks in a Sonos playlist. The underlying
+        call is quite complex as it both can move a track within the list or
+        delete a track from the playlist.  All of this depends on what tracks
+        and new_pos specify.
+
+        See the helper methods
+        :py:meth:`clear_sonos_playlist`, :py:meth:`move_in_sonos_playlist`,
+        :py:meth:`remove_from_sonos_playlist` for simplified usage.
```

Comment:
I would emphasise this even more. I think we need a Sphinx "Warning" that using this method, it is possible to mess up your playlist, if you are not very clear about the counting. You should only use this if you know what you are doing, and you probably want to use the helper methods instead!